mypy.ini

```diff
@@ -0,0 +1,18 @@
+[mypy]
+python_version = 3.6
+color_output = True
+error_summary = True
+
+check_untyped_defs = True
+
+disallow_untyped_calls = False
```

Comment:
Does flipping this to True make it emit many errors? If not, do this and address the errors; if yes, submit it as a separate PR.

Reply:
Turn it to True and there are no errors.

Reply:
So keep it true. The goal is to make mypy setup as strict as possible.

Reply:
OTOH you could use https://mypy.readthedocs.io/en/stable/config_file.html to make a follow-up PR.

Reply:
Should I add some other configs?

Reply:
Yes, but probably in another PR.

Reply:
Ok, I turned `disallow_untyped_calls` to True and will add other configs in another PR.